pkgs/development/tools/pnpm/fetch-deps/default.nix

```
@@ -29,6 +31,9 @@
             outputHash = "";
             outputHashAlgo = "sha256";
           };
+      extraPnpmInstallFlags =
+        if pnpmWorkspace != "" then "--filter=${pnpmWorkspace}"
+        else "";
```

```suggestion
      installFlags = lib.optional (pnpmWorkspace != "")  "--filter=${pnpmWorkspace}";
```
this would also be nice as similar to above, callers would be able to use standard `mkDerivation` variables rather than unique ones found here

---

> this would also be nice as similar to above, this would also be nice as similar to above, callers would be able to use standard `mkDerivation` variables rather than unique ones found here

I agree that `installFlags` is a better variable name (pushed this change), but I'm not sure how that helps someone override this variable with an interface similar to `mkDerivation`...